cmd/portal/send.go

```diff
@@ -12,18 +12,27 @@ import (
 	"github.com/spf13/viper"
 )
 
+// Set flags.
+func init() {
+	// Add subcommand flags (dummy default values as default values are handled through viper)
+	sendCmd.Flags().StringP("relay", "r", "", "address of the relay server")
```

Comment:
I think we should document the format which the relay flag expects more clearly. Maybe something like "address of the relay server (host:port)"?

Reply:
It does not necessarily have to be (host:port) right? How would you explain that? 🤔

Reply:
[2883d8d](https://github.com/SpatiumPortae/portal/pull/47/commits/2883d8d918bca049cc14c038a28e416a3cb88d27)